src/game.py

```diff
@@ -34,7 +34,7 @@ def __init__(self):
         self.infection_rate = None
         self.infection_rates = []
         self.epidemic_count = 0
-        self.diseases = {}
+        self.diseases = {} # TODO why is it a dict? Disease has colour property
```

Comment:
We use colour names as key labels to get an exact disease object e.g. when infecting cities (we use the colour of the drawn card).
Nothing to do here.